config/nephe.yml

```diff
@@ -453,6 +453,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - controlplane.antrea.io
+  resources:
+  - networkpolicies/status
+  verbs:
+  - create
+  - delete
```

Comment:
Do you need delete permission?

Reply:
not with this change, but when antrea supports delete on ANP realization, no change would be required on RBAC